mlir/include/mlir/Analysis/Presburger/Expr.h

```diff
@@ -0,0 +1,48 @@
+#ifndef MLIR_ANALYSIS_PRESBURGER_EXPR_H
+#define MLIR_ANALYSIS_PRESBURGER_EXPR_H
+
+#include "mlir/Analysis/Presburger/Set.h"
+
+namespace mlir {
+/// This class implements a data structure that represents a piecewise
```

Comment:
Presburger is already piecewise -- maybe something like "represents a Presburger expression, i.e., a piecewise affine expression"